lib/daun/version.rb

```diff
@@ -1,3 +1,3 @@
 module Daun
-  VERSION = '0.1.3'.freeze
```

Comment:
Nope. Will remove that

New versions of bundler refuse to install dependencies due to frozen String